exonum-java-binding/cryptocurrency-demo/pom.xml

```diff
@@ -224,6 +225,7 @@
           <systemPropertyVariables>
             <it.artifactFilename>${artifactFilename}.jar</it.artifactFilename>
             <it.artifactId>${artifactName}</it.artifactId>
```

Comment:
It clashes with the standard project.artifactId, I suggest some other name (it.exonumArtifactId?)